### PR TITLE
Make role more generic

### PR DESCRIPTION
**Pull request by lkiesow**
This patch makes the whole role more generic by allowing users to specify arbitrary configuration files instead of limiting access to just a few locations.

### CODE REVIEW

**Review by tibroc — APPROVED** [review]
looks good

File: tasks/main.yml [hunk 1]
```diff
@@ -7,15 +7,23 @@
 
 - name: Create configuration directories
```
Comment (anshockm) [inline tasks/main.yml:8]: - name:  Create TLS configuration directory

File: tasks/main.yml [hunk 2]
```diff
@@ -7,15 +7,23 @@
...
   ansible.builtin.file:
-    path: /etc/nginx/{{ item }}
+    path: /etc/nginx/tls/
     state: directory
     mode: '0755'
     owner: root
     group: root
+
+- name: Copy default configuration
```
Comment (anshockm) [inline tasks/main.yml:16]: - name: Copy TLS configration
or
- name: Copy default TLS configuation

File: tasks/main.yml (hunk 1 above)
Comment (tibroc) [reply tasks/main.yml:8]: Sorry, I just saw this too late. Do you mind opening a PR for this?

File: tasks/main.yml (hunk 2 above)
Comment (tibroc) [reply tasks/main.yml:16]: see above

**Pull request merged**